db/migrate/20240222034916_add_types_to_existing_memberships.rb

```diff
@@ -0,0 +1,25 @@
+class AddTypesToExistingMemberships < ActiveRecord::Migration[7.1]
+  class MigrationMember < ActiveRecord::Base
+    self.table_name = "members"
+    has_many :memberships, class_name: "MigrationMembership", foreign_key: "member_id"
+  end
```

Comment:
I like this pattern of defining a minimal model for the purposes of migration right inline. Helps protect against code collisions down the line. 👏🏻